src/repo/learnocaml_process_exercise_repository.ml

```diff
@@ -182,14 +194,23 @@ let main dest_dir =
                (fun acc id ->
                   all_exercises := id :: !all_exercises ;
                   from_file exercise_meta_enc (!exercises_dir / id / "meta.json")
-                  >>= fun (exercise_kind, exercise_stars) ->
+                  >>= fun ((exercise_kind, exercise_stars, exercise_identifier,
+                            author, focus, requirements,
+                            forward, backward), _) ->
                   let exercise_short_description = None in
                   let exercise =
                     read_exercise (!exercises_dir / id) in
                   let exercise =
                     { exercise_kind ; exercise_stars ;
                       exercise_title = Learnocaml_exercise.(get title) exercise ;
```

Comment:
Do you know why the exercise title is not part of the meta.json file? 

If this is the only place where the title.txt file is read, it could be a good idea to first look into meta.json for a title and if there is no such field, look into title.txt.

Reply:
I am not sure... maybe the `title.txt` is another remnant of the `FUN` platform, but I don't see a reason now to keep it separate indeed.

Reply:
Actually, there are no title.txt files in the FUN exercises (or at least, in the version I have) and the title is in the meta.json file.
An example :
{ "version": 1,
  "kind": "exercise",
  "title": "Displaying a Filesystem Hierarchy", 
  "stars": 3 }.

Reply:
Indeed, it might be a good idea to remove this file in this PR and merge it with `meta.json`. Another improvement would be the `short_description` field from `Learn_ocaml_index.exercise`, which is always set to `None` for the moment.

Reply:
Another file that could be easily integrated is `max_score.txt`, I do no see a good reason to keep it separated for the moment.

Reply:
@OCamlPro-Couderc Would you mind implement this feature if it is not already there, please? (The title in `meta.json` by default and in `title.txt` as a fallback.)

@lyrm Can you update the tutorial once the modification is done, please?

Reply:
@OCamlPro-Couderc Let's integrate `max_score.txt` to `meta.json` too.